cpp/include/cuspatial/experimental/iterator_factory.cuh

```diff
@@ -57,6 +59,18 @@ struct vec_2d_to_tuple {
   }
 };
 
+/**
+ * @internal
+ * @brief Helper to convert a `bouinding_box` into a tuple of elements
```

Comment:
```suggestion
 * @brief Helper to convert a `bounding_box` into a tuple of elements
```

Reply:
Good catch. Should really be just `box`.

```suggestion
 * @brief Helper to convert a `box` into a `tuple` of elements
```